blog: our open source tier is not competition (#1047)
flowforge/website: src/blog/2023/08/open-source-is-a-tier-not-competition.md

---
title: "Our Open Source offering is a tier, not our competition"
subtitle: 

> description: Often our customers think we're competing against 

Author: ```suggestion
description: Often our customers think we're competing against ourself, we're not
```

date: 2023-08-18
authors: ["zeger-jan-van-de-weg"]
image: "/blog/2023/08/images/matt-hardy-waves.jpg"
tags:
    - posts
    - flowfuse
---

More than once we’ve been in discussion with prospective customers on what tier
is the right tier for their current Node-RED adoption. The question is likely to
come up "Why wouldn't we just the open source version of FlowFuse?". The
implicit discussion created is one that is alike the question:
“Why wouldn’t we go with your competition?”. For FlowFuse, and most other open-core
companies like us, the open licensed and free to use core is tier, not competition.

<!--more-->

In the traditional sense, the prospective customer is right. By the definition,
a customer **buys** goods or services by exchanging it for **money**. For 
FlowFuse’s open-source edition, which is free as in beer and free as in speech,

> no money changes hands. It can be installed an run by anyone. Once the software

Reviewer: ```suggestion
no money changes hands. It can be installed and run by anyone. Once the software
```

> is being installed and used, For FlowFuse we consider we've gained a new customer.

Reviewer: ```suggestion
is being installed and used, we consider we've gained a new customer.
```

There’s an agreement in place, the Apache 2.0 license, and value is obtained by
the customer. The only missing component compared to an ‘ordinary’ sale is the lack
of money from the customer to FlowFuse. The fact that no monetary value is exchanged,
like the situation where a customer picks the competitor, doesn’t make the open

> source tier competition. Just a free tier.

Reviewer: ```suggestion
source tier competition. It is just a free tier.
```

Another reason it's really a tier is that the core of the product is the same. In

> many open-core products, the path to upgrade from the openly licenced product to

Reviewer: ```suggestion
many open-core products, the path to upgrade from the open source license product to
```

the paid tier is much alike customers are used to on SaaS models. In the reals of
self-managed software that's mostly uploading a license and  at times a few configuration
steps.

Furthermore, the open tier is a tier as the customer choses to not adopt all
capabilities. They're leaving value on the table. Either this is because it's not

> quite clear what the value is and if  the higher tiers provides

Reviewer: ```suggestion
quite clear what the value is or if  the higher tiers provides
```

> quite clear what the value obtained is and if the price for the higher tiers provides

Reviewer: ```suggestion
quite clear what the value is and if  the higher tiers provides
```

enough business value to warrant the expense. Or the adoption journey for the

> customer doesn't require the fully featured tiers yet.

Reviewer: ```suggestion
customer doesn't yet require the full featured tiers.
```

In

> FlowFuse’s case support is provided through our community forum and features

Reviewer: I think you can delete this last sentence. It is a bit out of context. If you want to keep it, I think it need to be reworded since it is not really clear. The first part is about support for community forum in our open source tier but the second part is about our enterprise features.

> The customer considers what they cannot adopt at the time, or indefinitely. In

Reviewer: I don't understand this sentence?

Author: ```suggestion
In
```

Trying to explain the decision is made anyway, there's no difference between buying a paid tier of OSS at this time. But it's only stressing it indeed.

> In
> FlowFuse’s case support is provided through our community forum and features
> like SSO/HA among others aren’t available either. 

Author: ```suggestion
```

What’s unique about open source software, is that customers can exchange value
towards the company and community building the software in other forms: by
opening issues, updating documentation, advocating for the OSS variant, among
other ways. While this is not **money**, it is significant for a young company
like FlowFuse.

## Challenges with a open core free tier

That's not to say that an open source tier is a silver bullet for a company. For
one, it's hard to track how many users a software package has, and who these users
are. For example; FlowFuse has [Telemetry](/docs/admin/telemetry/#usage-telemetry),
though it can be turned off. Nor do we know who hosts this software.

> Another challenge is around product segregation. At FlowFuse and beyond it's

Reviewer: ```suggestion
Another challenge is around product and feature packaging. At FlowFuse and other open core companies it's
```
